web/templates/shared/flashes.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= for flash_key <- [:info, :error] do %>
+  <%= if flash = get_flash(@conn, flash_key) do %>
+    <div class="flash flash-<%= flash_key %>">
+      <div class="container">
+        <p data-role="flash"><%= raw flash %></p>
```

Comment:
This came from another project. The reason we used the raw is that we often ended up wanting to show a link in the flash.